lib/resources/etc_group.rb

```diff
@@ -78,10 +78,12 @@ def where(conditions = {})
       }
       res = entries
 
-      conditions.each do |k, v|
-        idx = fields[k.to_sym]
-        next if idx.nil?
-        res = res.select { |x| x[idx].to_s == v.to_s }
+      unless res.nil?
```

Comment:
This is pretty concerning.  If entries is nil, then most of the resource is broken, and this resource does little to cushion the blow.  This fix will work for `inspec check`, but won't help people using `exec`; we're not exposing the issue, that the group file was not found.

```
inspec> describe etc_group('/no/such/file').where(group_name: 'awesome') do
inspec>   its('users') { should include 'bob' }
inspec> end

Profile: inspec-shell
Version: (not specified)

  #<Inspec::Resources::EtcGroupView:0x00007fe41a46d628>
     ×  users should include "bob"
     expected nil to include "bob", but it does not respond to `include?`

Test Summary: 0 successful, 1 failure, 0 skipped
inspec> inspec.version
=> "2.1.54"
inspec> exit
[cwolfe@lodi inspec-review]$ git log --oneline -n 1
a6039358 (HEAD -> chris-rock/etc-group-nil, origin/chris-rock/etc-group-nil) Fix case where res is nil in etc_group
```